en.html

```diff
@@ -65,11 +65,11 @@
                         </div>
                         <div class="col-md-5">
                             <div class="description text-center">
-                                <h2 class="reveal reveal-2">Optimize your industrial efficiency.</h2>
-                                <h5 class="reveal reveal-3"> Kami vous permet de suivre et d'optimiser votre production. </h5>
+                                <h2 class="reveal reveal-2">Improve your industrial efficiency.</h2>
+                                <h5 class="reveal reveal-3"> Kami let you track and your production efficiency. </h5>
```

Comment:
```suggestion
                                <h5 class="reveal reveal-3"> Kami let you track your production efficiency. </h5>
```
Je trouve cette phrase étrange avec le "and". Mais je vois ce que tu veux dire.
Sinon le reste rien à redire 👍